foundry/external_data_architectures.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import torch
 from torch.utils.data import Dataset
+from tensorflow.keras.utils import Sequence
 
 class FoundryDataset_Torch(Dataset):
```

Comment:
Underscores in class names is a little rare in Python. Maybe just call it "FoundryDataset" for Torch and "FoundrySequence" for TF? That way they match the nomenclature for the individual tools